### DISCUSSION TITLE
2.1.0

### DISCUSSION THREAD

**Discussion by caring-coder** [verronpro/office-stamper#387] [Announcements]
## What's Changed
* feat: Add v2.1.0 release notes to README by @caring-coder in https://github.com/verronpro/docx-stamper/pull/386
* Miscellaneous updates by @caring-coder in https://github.com/verronpro/docx-stamper/pull/361
** Update Readme howto to document the last API changes
** Update the mechanism to walk through a Word document
** First step to the separation of resolvers in two: the future 'engine resolver' (that will wrap the template features) and 'context resolver' (that will wrap the actual stamped data)
** Update exception management and messages
** The raw stamper does not carry any comment processors by default
** Bump dependencies org.springframework:spring-expression to 6.1.8
** Update documentation to be more Github-friendly

**Full Changelog**: https://github.com/verronpro/docx-stamper/compare/v2.0.1...v2.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/verronpro/docx-stamper/releases/tag/v2.1.0'>2.1.0</a>.</em>

**Comment by caring-coder** [comment]
Well, sonatype deployment seems to have some issues today. I'll keep trying tonight

**Reply by StefanLobbenmeierObjego** [reply]
looks like it never went through: https://mvnrepository.com/artifact/pro.verron.office-stamper/engine

maybe we will have more luck with 2.2.0 now?

**Reply by StefanLobbenmeierObjego** [reply]
yeah 2.2.0 is now - just received the renovate PR for it 😄
